ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -876,6 +876,21 @@ literals:
 - fire_alarm:
   - ACTIVE
   - INACTIVE
+- primary_fire_alarm: # better than using fire_alarm_1 (which will lead to fire_alarm_1_23)
+  - ACTIVE
+  - INACTIVE
+- secondary_fire_alarm:
+  - ACTIVE
+  - INACTIVE
+- primary_smoke_alarm: # for aspirating type smoke detectors
```

Comment:
same as question on fire alarm. describe please.

Reply:
Mainly to avoid having

`fire_alarm_1_23`

and instead have this:

`primary_fire_alarm_23`

Reply:
will now revert back to fire_alarm_1, fire_alarm_2 as they have separate sensors